src/containers/SemesterListContainer.jsx

```diff
@@ -0,0 +1,30 @@
+import { connect } from 'react-redux';
+import SemesterList from '../components/SemesterList';
+
+const shortenYear = (year) => {
+  const shortYearNumber = year % 100;
+  const shortYearString = shortYearNumber.toString();
+  if (shortYearString.length < 2) {
+    return '0' + shortYearString;
+  }
+  return shortYearString;
+};
+
+const makeSemesterList = (entryYear) => {
```

Comment:
I agree, but I chose to do it this way to reflect the information currently in the DB. Not sure if that is a group discussion or not...